packages/webapp/src/routes/__tests__/land/applicant-details-is-agent.spec.js

```diff
@@ -0,0 +1,81 @@
+import { submitGetRequest } from '../helpers/server.js'
+import constants from '../../../utils/constants.js'
+const url = constants.routes.APPLICANT_DETAILS_IS_AGENT
+
+describe(url, () => {
+  let viewResult
+  let h
+  let redisMap
+  let resultContext
+  let isApplicantAgent
+
+  beforeEach(() => {
+    h = {
+      view: (view, context) => {
+        viewResult = view
+        resultContext = context
+      },
+      redirect: (view, context) => {
+        viewResult = view
+      }
+    }
+
+    redisMap = new Map()
+    redisMap.set(constants.redisKeys.APPLICANT_DETAILS_IS_AGENT, 'yes')
+
+    isApplicantAgent = require('../../land/applicant-details-is-agent.js')
+  })
+
+  describe('GET', () => {
+    it(`should render the ${url.substring(1)} view`, async () => {
+      await submitGetRequest({ url })
+    })
+
+    it('should show correct party to be remove', async () => {
+      const request = {
+        yar: redisMap,
+        query: { orgId: '0' }
+      }
+
+      await isApplicantAgent.default[0].handler(request, h)
+
+      expect(viewResult).toEqual(constants.views.APPLICANT_DETAILS_IS_AGENT)
+    })
+  })
+
+  describe('POST', () => {
+    it('Should continue journey to APPLICANT_DETAILS_CONFIRM if yes is chosen and remove 1 legal party', async () => {
+      const request = {
+        yar: redisMap,
+        payload: { isApplicantAgent: 'yes' }
+      }
+
+      await isApplicantAgent.default[1].handler(request, h)
+
+      expect(viewResult).toEqual(constants.routes.APPLICANT_DETAILS_IS_AGENT)
+    })
+
+    it('Should continue journey to APPLICANT_ROLE_AS_LANDOWNER if no is chosen', async () => {
+      const request = {
+        yar: redisMap,
+        payload: { isApplicantAgent: 'no' }
+      }
+
+      await isApplicantAgent.default[1].handler(request, h)
+
+      expect(viewResult).toEqual(constants.routes.APPLICANT_DETAILS_IS_AGENT)
```

Comment:
Add TODO comment to refactor when required routing is available.

Reply:
I guess your comment is ment for no test above. This test when there is no answer will work now.

Reply:
done